src/engine/zsmOps.cpp

```diff
@@ -107,6 +107,16 @@ SafeWriter* DivEngine::saveZSM(unsigned int zsmrate, bool loop) {
     // TODO: incorporate the Furnace meta-command for init data and filter
     //       out writes to otherwise-unused channels.
   }
+  // Indicate the song's tuning as a sync meta-event
+  // specified in terms of how many 1/256th semitones
+  // the song is offset from standard A-440 tuning.
+  // This is mainly to benefit visualizations in players
+  // for non-standard tunings so that they can avoid
+  // displaying the entire song held in pitch bend.
+  // Tunings offsets that exceed a half semitone
+  // will simply be represented in a different key
+  signed char tuningoffset=(signed char)round(3072*(log(song.tuning/440.0)/log(2)));
```

Comment:
Handle overflow?

Reply:
The implicit overflow here should already do the right thing.  I could add an `&0xff` to make the intent clearer

Reply:
Got it